contracts/dynamic-caller-contract/src/contract.rs

```diff
@@ -24,8 +24,8 @@ struct CalleeContract {
     address: Addr,
 }
 
-// When `user_defined_mock = true`, user can and must write mock implement
-// of the trait for specified struct with `#[cfg(not(target_arch = "wasm32"))]`.
+// When a trait is specified with `user_defined_mock = true`, user can and must write mock implement for test.
```

Comment:
This is not a spec of traits, but a spec of dynamic_link. This comment looks confusing.

Reply:
I fixed as pointed out.